timemachine/fe/chiral_utils.py

```diff
@@ -231,12 +231,38 @@ def apply_mapping(c, i, j, k):
     return conflicts
 
 
+def _has_chiral_atom_map_flips_one_direction(
+    core: np.ndarray,
+    chiral_set_a: ChiralRestrIdxSet,
+    chiral_set_b: ChiralRestrIdxSet,
+) -> bool:
+    # _find_atom_map_chiral_conflicts_one_direction, except (1) return bool not set, (2) hard-code mode = FLIP
```

Comment:
nit: Add a comment why you don't just return bool(_find_atom_map_chiral_conflicts_one_direction(...))` 

I assume this is for performance reasons to fail quickly

Reply:
Separate implementation for the case where we only need a bool, in case we want to apply performance optimizations specific to this case, e.g. as suggested in https://github.com/proteneer/timemachine/pull/908#pullrequestreview-1176448612 and https://github.com/proteneer/timemachine/pull/908#issuecomment-1314425524